cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Babbage/Config/MigrateConsumedPruneTxOut.hs

```diff
@@ -316,15 +293,12 @@ noPruneSameBlock =
     assertBlockNoBackoff dbSyncEnv 98
     assertEqQuery dbSyncEnv DB.queryTxOutConsumedCount 0 "Unexpected TxOutConsumedByTxId after rollback"
   where
-    cmdLineArgs =
-      initCommandLineArgs
-        { claConfigFilename = "test-db-sync-config-consumed.json"
-        }
+    cmdLineArgs = initCommandLineArgs
     testLabel = "configNoPruneSameBlock"
 
 migrateAndPruneRestart :: IOManager -> [(Text, Text)] -> Assertion
 migrateAndPruneRestart = do
-  withCustomConfig cmdLineArgs Nothing babbageConfigDir testLabel $ \interpreter mockServer dbSyncEnv -> do
+  withCustomConfig cmdLineArgs (Just configTxOutPruneForceTxIn) babbageConfigDir testLabel $ \interpreter mockServer dbSyncEnv -> do
```

Comment:
I think you need a different config fucntion there. It was not prune before. Same for some other cases as well.